packages/cli/src/DebugInfo.ts

```diff
@@ -0,0 +1,145 @@
+import Debug from '@prisma/debug'
+import type { Command } from '@prisma/internals'
+import { arg, format, getConfig, getSchemaPath, HelpError, isError, isCi, isInteractive } from '@prisma/internals'
+import { bold, dim, red, underline } from 'kleur/colors'
+import fs from 'fs'
+import { getRootCacheDir } from '../../fetch-engine/src/utils'
+
+const debug = Debug('prisma:cli:debug')
+
+/**
+ * $ prisma debug
+ */
+export class DebugInfo implements Command {
+  static new(): DebugInfo {
+    return new DebugInfo()
+  }
+
+  private static help = format(`
+  Print information helpful for debugging and bug reports
+
+  ${bold('Usage')}
+
+    ${dim('$')} prisma debug [options]
+
+  ${bold('Options')}
+
+    -h, --help     Display this help message
+    --schema       Custom path to your Prisma schema
+`)
+
+  async parse(argv: string[]): Promise<string | Error> {
+    const args = arg(argv, {
+      '--help': Boolean,
+      '-h': '--help',
+      '--schema': String,
+      '--telemetry-information': String,
+    })
+
+    if (isError(args)) {
+      return this.help(args.message)
+    }
+
+    if (args['--help']) {
+      return this.help()
+    }
+
+    const formatEnvValue = (name: string) => {
+      const value = process.env[name]
+
+      if (value === undefined) {
+        return dim(`- ${name}:`)
+      }
+      return bold(`- ${name}: \`${value}\``)
+    }
+
+    const schemaPath = await getSchemaPath(args['--schema'])
+    const rootCacheDir = await getRootCacheDir()
+
+    return `${underline('-- Prisma schema --')}
+Path: ${schemaPath}
+
+${underline('-- Local cache directory for engines files --')}
+${rootCacheDir}
+
+${underline('-- Environment variables --')}
+When not set, the line is dimmed and no value is displayed.
+When set, the line is bold and the value is inside the \`\` backticks.
+For more information, see https://www.prisma.io/docs/reference/api-reference/environment-variables-reference
```

Comment:
Might be worth to split the Prisma specific ones from the more general ones? The link here only applied to the specific ones.

(I think the "For general debugging" and "For proxy settings" would make it into the "general" section, everything else seems Prisma specific)

Reply:
done!